x-pack/plugin/core/template-resources/src/main/resources/ecs@mappings.json

```diff
@@ -23,6 +23,26 @@
             "unmatch_mapping_type": "object"
           }
         },
+        {
+          "ecs_event_original_non_indexed_keyword": {
+            "mapping": {
+              "type": "keyword",
+              "index": false,
+              "doc_values": false
+            },
+            "path_match": "event.original"
```

Comment:
Can we make this an array like we have here? https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/template-resources/src/main/resources/ecs%40mappings.json#L19 Like this the overall template stays simpler and is easy to extend.